package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "digitalmarketplace-frontend-toolkit",
   "description": "A toolkit for design patterns used across Digital Marketplace projects",
-  "version": "28.13.2",
+  "version": "28.14.0",
```

Comment:
Meh, now that I think about it we probably should have done this as a breaking change to force/encourage people to move over to using the toolkit's new base layout.